netbench/cdk/lambda/exportS3.js

```diff
@@ -0,0 +1,19 @@
+const AWS = require('aws-sdk')
+const cloudconfig = {
+  apiVersion: '2014-03-28',
+  region: process.env.REGION, 
+}
+const cloudwatchlogs = new AWS.CloudWatchLogs(cloudconfig)
+exports.handler =  async (event, context) => {
+   const params = {
+    destination: process.env.BUCKET_NAME, 
+    from: new Date().getTime() - 600000, //ten minutes, will add exact time stamp in future iteration
```

Comment:
What do you mean by exact time stamp? Also, when you have a task like this that you plan to address in a future PR, its better to put `TODO` in the comment so we know there is more work to be done

Reply:
In the future, I'll be using the timestamp of the start of state machine execution once the server is also moved into the state machine. This will give a more tight and accurate window to pull logs from. I'll leave a TODO comment explaining this.